kasa/smart/smartdevice.py

```diff
@@ -22,6 +22,8 @@
 if TYPE_CHECKING:
     from .smartmodule import SmartModule
 
+PARENT_ONLY_MODULES = [DeviceModule, TimeModule, Firmware, CloudModule]  # noqa: F405
```

Comment:
Maybe add a quick comment about what's this is and how the modules are chosen/when this should be updated?

Reply:
Done